edge-hub/test/Microsoft.Azure.Devices.Edge.Hub.Mqtt.Test/UsernameParserTest.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Hub.Core.Test
+{
+    using System.Collections.Generic;
+    using Microsoft.Azure.Devices.Edge.Hub.Mqtt;
+    using Microsoft.Azure.Devices.Edge.Util.Test.Common;
+    using Xunit;
+
+    [Unit]
+    public class UsernameParserTest
```

Comment:
nit: to be more precise, this file and tests should be named `MqttUsernameParserTest`

Reply:
changed the class name